Doc/library/urllib.request.rst

```diff
@@ -218,6 +218,7 @@ The following classes are provided:
    (X11; U; Linux i686) Gecko/20071127 Firefox/2.0.0.11"``, while
    :mod:`urllib`'s default user agent string is
    ``"Python-urllib/2.6"`` (on Python 2.6).
+   Note that all header keys are sent in Camel case.
```

Comment:
```suggestion
   All header keys are sent in camel case.
```
"Note that" is redundant, and "camel" should be lower case because it's not a proper noun. (If you're not aware, it's called that because it looks like a camel's two humps.)